README.md

```diff
@@ -1,14 +1,12 @@
 # Chrono-TZ
 
-`Chrono-TZ` is a library that provides implementors of the
-[`TimeZone`][timezone] trait for [`rust-chrono`][chrono]. The
-impls are generated by a build script using the [`IANA database`][iana]
-and [`parse-zoneinfo`][parse_zoneinfo].
+`Chrono-TZ` is a library that provides implementors of the [`TimeZone`] trait for [chrono]. The
```

Comment:
Please don't reflow if the commit message is "Fix dead links"...

Reply:
It didn't feel significant enough for a separate commit... I adjusted the commit message. Is that okay?